lib/wda/webdriveragent.js

```diff
@@ -375,9 +394,24 @@ class WebDriverAgent {
     this.webDriverAgentUrl = this.url.href;
   }
 
-  async quitAndUninstall () {
+  /**
+   * Quit running WDA and uninstall it.
+   * @param {?string} updatedWDABundleId
+   */
+  async quitAndUninstall (updatedWDABundleId) {
+    let installedBundleIdentifier = updatedWDABundleId || WDA_BUNDLE_ID;
+    const status = await this.getStatus();
```

Comment:
can getStatus throw an exception?

Reply:
No. It returns `null` if an exception happened in the method.